products/sle/main.pm

```diff
@@ -443,7 +443,12 @@ sub load_boot_tests() {
         }
     }
     elsif (uses_qa_net_hardware) {
-        loadtest "installation/qa_net.pm";
+        if (get_var("BEIJING")) {
```

Comment:
I am pretty sure this variable should be replaced by something reasonable, comments by others?